README.md

```diff
@@ -13,42 +13,50 @@ This project contains core logic and the utilities for the APIMatic's Java SDK
+| [`HttpRequest`](./src/main/java/io/apimatic/core/HttpRequest.java)                                                               | An HTTP request is made by a client, to a named host, which is located on a server                                                                                                                                                                    |
+| [`ResponseHandler`](./src/main/java/io/apimatic/core/ResponseHandler.java)                                                       | Handler that encapsulates the process of generating a response object from a Response                                                                                                                                                                 |
+| [`SdkLogger`](./src/main/java/io/apimatic/core/logger/SdkLogger.java)                                                            | A class to log Request and Response.                                                                                                                                                                                                                  |
+| [`NullSdkLogger`](./src/main/java/io/apimatic/core/logger/NullSdkLogger.java)                                                    | Null SdkLogger implementation where logger is not configured.                                                                                                                                                                                         |
+| [`Slf4jLogger`](./src/main/java/io/apimatic/core/logger/Slf4jLogger.java)                                                        | Logger implementation supporting Slf4j Facade.                                                                                                                                                                                                        |
+| [`ConsoleLogger`](./src/main/java/io/apimatic/core/logger/ConsoleLogger.java)                                                    | Default Logger implementation if logging is enabled.                                                                                                                                                                                                  |
+| [`LoggerFactory`](./src/main/java/io/apimatic/core/logger/LoggerFactory.java)                                                    | Returns an instance of ApiLogger based on the provided configuration.                                                                                                                                                                                 |
```

Comment:
The link for LoggerFactory is broken and should be fixed.
![image](https://github.com/apimatic/core-lib-java/assets/18752818/938f579b-9e07-47ed-b2ec-6bba16467d3c)

Reply:
Broken link has been resolved